packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/import/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dedicatedCloud_USER_import_title": "Importer un utilisateur ou un groupe depuis votre Active Directory",
+  "dedicatedCloud_USER_import_content": "Veuillez renseigner les champs suivants pour importer dans votre Hosted Private Cloud un utilisateur ou un groupe de votre Active Directory.",
+  "dedicatedCloud_USER_import_select_federation_placeholder": "Veuillez choisir l'Active Directory concerné",
+  "dedicatedCloud_USER_import_user_exemple": "utilisateur",
+  "dedicatedCloud_USER_import_group_exemple": "groupe",
+  "dedicatedCloud_USER_import_radio_user": "Importer un utilisateur",
+  "dedicatedCloud_USER_import_radio_group": "Importer un groupe",
+  "dedicatedCloud_USER_import_field_label_userName": "Nom utilisateur",
+  "dedicatedCloud_USER_import_field_label_groupName": "Nom du groupe",
+  "dedicatedCloud_USER_import_field_mandatory": "Champ obligatoire",
+  "dedicatedCloud_USER_import_button_close": "Fermer",
+  "dedicatedCloud_USER_import_error": "L'import du nouvel utilisateur n'a pas pu être effectué."
```

Comment:
i see the error message, here we don't need the success message?

Reply:
It is not necessary as the status is present in the "pollTaskTraker" which returns "done". When we then click on "close" the page refreshes and displays the new user.